data_pulling_utils/json_to_csv.py

```diff
@@ -0,0 +1,34 @@
+from os import walk, path, remove
```

Comment:
If this is a separate utility and can generically convert any json to a csv then I'll give it a name that explains that.. `json-to-csv` or `json-csv-converter` or similar.

If this is really tied to data_pulling / metrics utility I'd move it in the same folder.